stacks/openshift/application/oidc-authservice-for-appid/README.md

```diff
@@ -0,0 +1,19 @@
+# oidc-authservice for IBM Cloud AppID
+
+## Prerequisites
+
+* Provisioning an AppID instance from IBM Cloud. the free tier works.
+* keep the configuration parameters including:
```

Comment:
Retrieve the following configuration parameters from your AppID. They will be used to configure the OIDC auth service.